docs/compose/treafik/docker-compose.yml

```diff
@@ -0,0 +1,139 @@
+version: '2'
+
+services:
+
+  # This would normally not be here, but where you define your system services
+  traefik:
+    image: traefik:alpine
+    command: --docker
+    restart: always
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+      - "/data/traefik/acme.json:/acme.json"
+      - "/data/traefik/traefik.toml:/traefik.toml"
+      # This may be needed (plus defining mailu_default external: true) if traefik lives elsewhere
+      # networks:
+      # - mailu_default
+
+  # The certdumper is external, but adapted for using traefik with mailu
+  certdumper:
+    restart: always
+    image: nebukadneza/traefik-certdumper:latest
+    environment: # Make sure this is the same as the main=-domain in traefik.toml
+      - DOMAIN=mail.your.doma.in
+    volumes:
+      - "/data/traefik:/traefik"
+      - "$ROOT/certs:/output"
+
+  front:
+    image: mailu/nginx:$VERSION
+    restart: always
+    env_file: .env
+    logging:
+      driver: $LOG_DRIVER
+    labels: # Traefik labels for simple reverse-proxying
+      - "traefik.enable=true"
+      - "traefik.port=80"
+      - "traefik.frontend.rule=Host:mail.your.doma.in"
+      - "traefik.docker.network=mailu_default"
+    ports:
+      - "80" # Let’s not expose 80 or 443 on host, since that’s taken by traefik
+      - "$BIND_ADDRESS4:110:110"
+      - "$BIND_ADDRESS4:143:143"
+      - "$BIND_ADDRESS4:993:993"
+      - "$BIND_ADDRESS4:995:995"
+      - "$BIND_ADDRESS4:25:25"
+      - "$BIND_ADDRESS4:465:465"
+      - "$BIND_ADDRESS4:587:587"
+      - "$BIND_ADDRESS6:80:80"
+      - "$BIND_ADDRESS6:443:443"
+      - "$BIND_ADDRESS6:110:110"
+      - "$BIND_ADDRESS6:143:143"
+      - "$BIND_ADDRESS6:993:993"
+      - "$BIND_ADDRESS6:995:995"
+      - "$BIND_ADDRESS6:25:25"
+      - "$BIND_ADDRESS6:465:465"
+      - "$BIND_ADDRESS6:587:587"
+    volumes:
+      - "$ROOT/certs:/certs" # Mount both certs directory (for dhparams.pem) and your domains key
+      - "$ROOT/overrides/nginx:/overrides"
+      - /data/traefik/ssl/mail.your.doma.in.crt:/certs/cert.pem
+      - /data/traefik/ssl/mail.your.doma.in.key:/certs/key.pem
```

Comment:
Instead of using `mail.your.doma.in.key` and `mail.your.doma.in.crt` you could define a variable called `TRAEFIK_HOST` and do something like:
```
      - /data/traefik/ssl/$TRAEFIK_HOST.crt:/certs/cert.pem
      - /data/traefik/ssl/$TRAEFIK_HOST.key:/certs/key.pem
```
It's bit cleaner and people can just add it to their `.env` or just change the variable to a hard coded string. (However they wish).